setup/module/api/user_data.sh

```diff
@@ -1,12 +1,11 @@
-# The majority of this script was largely taken from AWS's documentation
-# on using CloudWatch logs with container instances
-# https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_cloudwatch_logs.html
-
 Content-Type: multipart/mixed; boundary="==BOUNDARY=="
 MIME-Version: 1.0
 
 --==BOUNDARY==
 #!/bin/bash
+# The majority of this script was largely taken from AWS's documentation
```

Comment:
Seems like you forgot to add the actual changes to the script(s)?

Reply:
The change was to move this commented section below the declaration of the multipart/mixed Content-Type. I suppose I could move it above the `--==BOUNDARY==` but the comment should still be relevant for the bash script itself, and thus why I have opted to comment there.

It appeared in my testing that having this commented section above the Content-Type line caused an issue where the script would not execute and would not be able to register itself as an ECS cluster instance.